asapdiscovery-ml/asapdiscovery/ml/inference/inference.py

```diff
@@ -173,8 +176,43 @@ def predict(self, g: dgl.DGLGraph):
         g : dgl.DGLGraph
             DGLGraph object.
 
+        Returns
+        -------
+        np.ndarray
+            Predictions for each graph.
         """
         with torch.no_grad():
             output_tensor = self.model(g, g.ndata["h"])
+            # unsure exactly why this reshape is done
```

Comment:
this was done in the training methods to match the shape of the labels, probably isn't needed here